Formula/elasticsearch.rb

```diff
@@ -34,6 +35,10 @@ def install
     # Install everything else into package directory
     libexec.install "bin", "config", "lib", "modules"
 
+    inreplace libexec/"bin/elasticsearch-env",
+              "if [ -z \"$ES_PATH_CONF\" ]; then ES_PATH_CONF=\"$ES_HOME\"/config; fi",
```

Comment:
needs to say <= 80 character

Reply:
I don't understand this and I actually find it quite frustrating. Either the rule is 80 characters or it's what is enforced by audit. Please, enforcing on contributors in PR reviews rules different than what audit enforces makes maintaining these formula more onerous than necessary. I'm with you: you're a maintainer, you have every right to request changes to the formula to have consistency across formula (e.g., what methods are used, etc.) yet if you have a tool that enforces a certain code style than that should be sufficient for PR reviews and contributors should not have to deal with formatting differently than what is necessary to get audit to pass.

Reply:
@jasontedor as I said above, there are rules that we apply to the maximum extent possible that cannot be enforced programmatically without causing CI havoc. Currently the actually enforced maximum is 190 but we will stay <= 80 where possible.

Reply:
I think this is a mistaken approach, it's frustrating to deal with as a contributor.

Reply:
@jasontedor OK. We will endure the _heinous_ line length if it makes you :smile_cat:

Reply:
@ilovezfs ❤️

Reply:
My recommendation would be this:
 - if 80 characters is what you want to enforce, reduce the limit in audit to 80 characters
 - whitelist all formula that currently have a line exceeding the 80 character limit; this keeps new formula and existing formula that do not have lines exceeding the limit from breaching the limit
 - warn but do not error on formula that are whitelisted and have lines that exceed the 80 character limit so that maintainers are encouraged to bring them under the limit
 - prune formula from the whitelist that are corrected to have all lines under the 80 character limit (e.g., a CI job that checks the whitelisted formula on a periodic basis)
 - add the ability to annotate a line as passing the audit check even if it exceeds 80 characters if  not exceeding 80 characters really is unavoidable for a given line